crates/worker-proto/proto/nemo/relay/worker/v1/plugin_worker.proto

```
@@ -61,6 +67,62 @@ enum RegistrationSurface {
   SCOPE_SANITIZE_END_GUARDRAIL = 32;
 }
 
+enum RuntimeRegistrationOwnerKind {
+  RUNTIME_REGISTRATION_OWNER_KIND_UNSPECIFIED = 0;
+  CORE = 1;
```

Buf (1.72.0)

[error] Enum value name "CORE" should be prefixed with "RUNTIME_REGISTRATION_OWNER_KIND_".

(ENUM_VALUE_PREFIX)